README.md

```diff
@@ -94,15 +94,14 @@ the conf file will be mounted into docker container at runtime to PGSERVICEFILE
 
 To active https with LetsEncrypt just follow the following instructions:
```

Comment:
Currentrly I am currently unable to verify the correctness of these statements.

Reply:
@Raruto I just check and it works fine.